cpp/models/abm/parameters.h

```diff
@@ -716,27 +719,6 @@ class Parameters : public ParametersBase
             }
         }
 
-        if (this->get<MaskProtection>()[MaskType::Community] < 0.0 ||
```

Comment:
Instead of checking these fix values, we could check for some reasonable time point arguments to the function if the result is between 0 and 1. Just an idea though for you to think about.